modules/standard/Math.chpl

```diff
@@ -46,6 +46,37 @@ module Math {
   private use CTypes;
   private use AutoMath;
 
+  //////////////////////////////////////////////////////////////////////////
+  // Constants (included in chpldocs)
+  //
+
+  /* e - exp(1) or  the base of the natural logarithm */
+  param e = 2.7182818284590452354;
+  /* log2(e) */
+  param log2_e = 1.4426950408889634074;
```

Comment:
I am not up-to-date, did we stabilize names with underscores?

Reply:
There's a couple of things at play here:
- some of these would then share a name with functions
- it may be too late to do some drastic naming shifts for the Math module specifically